### PR TITLE
Personal variables order catalog

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/endpoint-reference/endpoint-variable.rst [hunk 1]
```diff
@@ -401,6 +401,53 @@ Behaves sames as PATCH.
...
 .. _endpoint-variables-weights:
 
+
+Personal Variable Order Catalog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``/datasets/{id}/variables/personal/variables/``
+
+The personal variable order comes accompanied with a catalog containing
+a variables present in the order's graph. This should provide access to
```
Comment (dianaclarke) [inline source/endpoint-reference/endpoint-variable.rst]: s/a variables/the variables/

File: source/endpoint-reference/endpoint-variable.rst [hunk 2]
```diff
@@ -401,6 +401,53 @@ Behaves sames as PATCH.
...
+metadata of those variables.
```
Comment (dianaclarke) [inline source/endpoint-reference/endpoint-variable.rst]: Perhaps replace:

- `This should provide access to metadata of those variables.`

with:

- `This provides access to the metadata for those variables.`

File: source/endpoint-reference/endpoint-variable.rst (hunk 1 above)
Comment (jjdelc) [reply source/endpoint-reference/endpoint-variable.rst]: yes

File: source/endpoint-reference/endpoint-variable.rst (hunk 2 above)
Comment (jjdelc) [reply source/endpoint-reference/endpoint-variable.rst]: Done

**Review by dianaclarke — APPROVED** [review]

**Pull request merged**